init.lua

```diff
@@ -236,5 +236,14 @@ function M.audio(data, opts)
   return win
 end
 
+function M.html(html, opts)
+  opts = opts or {}
+  local win = opts.win or uid()
+
+  send({ command='html', id=win, title=opts.title, html=mime.b64(html) })
```

Comment:
i was trying to not have escape characters get in the way. i think i can remove the .b64, i'll try and get back on this after making sure that there are no side-effects.